### PR TITLE
Ping360 control

### PR DESCRIPTION
**Pull request by jaxxzer**
Update control panel to use sliders, fix some bugs. Needs #563 

Sliders require the least amount of user interaction to adjust each of these settings, and this is more mobile friendly (in case we support mobile in the future).

### CODE REVIEW

**Comment by jaxxzer** [comment]
Screenshot_20190722_085313

**Comment by Williangalvani** [comment]
I found a few issues:
The sector is messed up, it is in gradians but the maximum and step are degrees. It needs the 200/180 factor.

Screenshot from 2019-07-22 07-19-03

The range seems to be off by one:
Screenshot from 2019-07-22 07-09-44

Can we have .5 step for the range? It is locking to .99 a lot, I don't know how the range labels deal with that. I also couldn't get to 100, only 99.99.

**Comment by jaxxzer** [comment]
> Can we have .5 step for the range? It is locking to .99 a lot, I don't know how the range labels deal with that. I also couldn't get to 100, only 99.99.

Do you want to help me with this? This kind of thing is time consuming for me. I was planning on letting @patrickelectric deal with it when he gets back. Right now I am trying to make the core functionality actually function. I am passing these fine details for later.

I think 1.0 step for range is fine, but we can try 0.5 if you like.

**Comment by Williangalvani** [comment]
> Do you want to help me with this?

Sure! I'll take a look at these smaller things.

**Comment by jaxxzer** [comment]
I've fixed some of them already with a force push. The polar grid/range rings/text labels needs an overhaul too. The way the labels are calculated right now is more round-about than I want it to be.

**Comment by jaxxzer** [comment]
> The sector is messed up, it is in gradians but the maximum and step are degrees. It needs the 200/180 factor.

done

**Comment by kklemens** [comment]
Angle Offset makes more sense to be a dial. This would fit the theme:
Origin Dial

**Comment by brianhBR** [comment]
Frequency should default to 750 kHz.

**Comment by brianhBR** [comment]
Gain setting should be "Receiver Gain" and the values should be Low, Normal, and High instead of numerical values.

File: qml/Ping360ControlPanel.qml [hunk 1]
```diff
@@ -23,168 +23,80 @@ Item {
...
-                    }
-                }
+            PingSlider {
+                text: "Sector Angle (degrees)"
+                value: ping.sectorSize
+                control.from: 30
+                control.to: 360
+                control.stepSize: 15
+                control.snapMode: Slider.SnapAlways
+                // TODO: fix calculation
```
[the remark this answers is not in the document]
Comment (jaxxzer) [reply qml/Ping360ControlPanel.qml]: nice :eye:

**Pull request merged**